scripts/tap-generate-docs/src/tap_generate_docs.clj

```diff
@@ -21,6 +21,13 @@
   (or (= {} property-json-schema-partial)
       (string? (property-json-schema-partial "type"))))
 
+(defn type-declaration?
+  "A valid type declaration is either a string or a list. Never a map,
+   which covers the rest of JSON schema."
+  [sub-schema]
+  (or (seq? sub-schema)
```

Comment:
I'm unsure if this will do exactly what you think it will do. 

```
user=> (seq? [:foo :bar])
false
user=> (seq? {:foo :bar})
false
user=> (seq? (seq [:foo :bar]))
true
user=> (seq? (seq {:foo :bar}))
true
```

I think instead you want `sequential?`

```
user=> (sequential? {:foo :bar})
false
user=> (sequential? [:foo :bar])
true
```

but be cautious that would be true for anything that was mapped over.

```
user=> (sequential? (map identity {:foo :bar}))
true
```